pychunkedgraph/backend/chunkedgraph.py

```diff
@@ -1589,6 +1589,55 @@ def read_log_rows(self, operation_ids: Optional[Sequence] = None):
 
         return log_records_d
 
+    def read_log_rows_for_timespan(self, start_time: Optional[datetime.datetime] = None,
```

Comment:
That looks good, it should combined with the existing function though / this here should be the new function since it appears to be the existing function + time range.